components/bpmn/org.wso2.carbon.bpmn.uuf.ui.stub/src/main/java/org.wso2.carbon.bpmn.uuf.ui/Activator.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.bpmn.uuf.ui;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.carbon.ui.bpmn_explorer.service.BPMNExplorerService;
+import org.wso2.carbon.ui.bpmn_explorer.service.BPMNExplorerServiceImpl;
+
+public class Activator implements BundleActivator {
+
+    private static final Logger log = LoggerFactory.getLogger(Activator.class);
+
+    /**
+     * Implements BundleActivator.start().
+     *
+     * @param context the framework context for the bundle.
+     */
+    public void start(BundleContext context) throws Exception {
+        context.registerService(BPMNExplorerService.class.getName(), new BPMNExplorerServiceImpl(), null);
+        if (log.isDebugEnabled()) {
+            log.debug("BPMN-Explorer Bundle Started.");
+        }
+    }
+
+    /**
+     * Implements BundleActivator.stop().
+     *
+     * @param context the framework context for the bundle.
+     */
+    public void stop(BundleContext context) {
+        if (log.isDebugEnabled()) {
```

Comment:
Be careful when copy paste :). You are printing "bundle started" when stopping also. 
Another thing is when you have no string manipulations in debug messages, you don't need to check isDebugEnabled(). It is an extra if.